documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -67,14 +67,15 @@ EOF
 <1> The `ownerReferences` section is optional.
 <2> Specify the type of source provider. Allowed values are `ovirt`, `vsphere`, `openstack`, `ova`, and `openshift`. This label is needed to verify the credentials are correct when the remote system is accessible and, for {rhv-short}, to retrieve the {manager} CA certificate when a third-party certificate is specified.
 <3> The `stringData` section for OVA is different and is described in a note that follows the description of the `Secret` manifest.
-<4> Specify the vCenter user, the {rhv-short} {manager} user, or the {osp} user.
+<4> Specify the vCenter user, the {rhv-short} {manager} user, or the {osp} user. For {ocp-short}, instead of entering a `user`, create a service account and a token, as described in link:https://access.redhat.com/solutions/7025261[How to create a long lived service account token in RHOCP]. This label is not used for OVA.
```

Comment:
it makes sense to explain how to get a proper token but I'm not sure it's the best place to do that since we also instruct how to specify an OCP token in the `4.4.2.1. Adding an OpenShift Virtualization destination provider` subsection - there we say in the prerequisites part: `You must have an OpenShift Virtualization service account token with cluster-admin privileges.`. maybe we should have prerequisites here as well and add that information in both places, what do you think?

also, that's not a label (see another comment on that below)